src/SearchInput/__stories__/__snapshots__/SearchInput.stories.storyshot

```diff
@@ -158,6 +157,7 @@ exports[`Storyshots Component API/SearchInput Disabled 1`] = `
               >
                 <div
                   className="fd-input-group--control fd-input-group"
+                  onClick={[Function]}
```

Comment:
Is it right for the onClick to be on a different element than the focus and blur?